creator/studies/migrations/0030_add_and_populate_program.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 2.2.24 on 2021-11-02 19:19
+
+"""
+This migration creates the program attribute for the Study model and also
+populates it using the spreadsheet found in the migrations folder.
+"""
+
+
+from django.db import migrations, models
+
+import pandas as pd
+import os
+
+
+def set_program(apps, schema_editor):
+    """
+    Populate program for all Study objects using the provided data.
+    """
+    df = pd.read_csv(
+        os.path.abspath("creator/studies/migrations/study_metadata.csv")
+    )
+    programs = pd.Series(df.program.values, index=df.kf_id).to_dict()
+    Study = apps.get_model("studies", "Study")
+
+    for study in Study.objects.all():
+        kf_id = study.kf_id
+        if kf_id in programs:
+            study.program = programs[kf_id].strip()
+            study.save()
+
+
+def undo_set_program(apps, schema_editor):
+    """
+    Set program to the Organization name for all Studies.
+    """
+    Study = apps.get_model("studies", "Study")
+    for study in Study.objects.all():
+        if study.organization:
```

Comment:
I don't think this makes sense here, since this is supposed to undo setting the program. Shouldn't all study programs just be set to null/None?